<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCancelList">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>760</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnSearch">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm kiếm:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1181</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMonth">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="tableMonth">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1181</width>
        <height>671</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Mã thẻ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Biển số</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loại xe</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loại vé</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày đăng ký</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày hết hạn</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDay">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tableDay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1181</width>
        <height>671</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Mã thẻ</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Biển số</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loại xe</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Loại vé</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày đăng ký</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ngày hết hạn</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1184</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
